03-rnaseq/00-intro-to-rnaseq.Rmd

```diff
@@ -6,3 +6,125 @@ output:
     toc: true
     toc_float: true
 ---
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Introduction to RNA-seq technology](#introduction-to-rna-seq-technology)
+  - [RNA-seq data **strengths**:](#rna-seq-data-strengths)
+  - [RNA-seq data **limitations**:](#rna-seq-data-limitations)
+  - [About quantile normalization](#about-quantile-normalization)
+  - [More resources on RNA-seq technology:](#more-resources-on-rna-seq-technology)
+- [About DESeq2](#about-deseq2)
+  - [DESeq2 objects](#deseq2-objects)
+  - [DESeq2 normalization methods](#deseq2-normalization-methods)
+  - [Further resources for DESeq2](#further-resources-for-deseq2)
+    - [Why doesn't the gene I care about show up in this dataset?](#why-doesnt-the-gene-i-care-about-show-up-in-this-dataset)
+    - [Why do these examples use DESeq2 and not EdgeR or ____](#why-do-these-examples-use-deseq2-and-not-edger-or-____)
+    - [What if I care about isoforms?](#what-if-i-care-about-isoforms)
+- [References](#references)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction to RNA-seq technology
+
+Data analyses are generally not "one size fits all"; this is particularly true as we are working with different technologies. 
+This tutorial has example analyses [organized by technology](../01-getting-started/getting-started.html#about-how-this-tutorial-book-is-structured) so you can follow examples that are more closely tailored to the nature of the data at hand. 
+
+As with all experimental methods, RNA-seq has strengths and limitations that you should consider in regards to your scientific questions. 
+
+### RNA-seq data **strengths**:  
+
+- RNA-seq can collect data on more transcripts (it is less bound to a pre-determined set of probes like microarray is). 
+- It's values are considered more dynamic than microarray values which are constrained to the number of probes.
+
+### RNA-seq data **limitations**:  
```

Comment:
Very minor (petty?), but I find the inconsistent usage of `:` at the end of headers to be distracting.